inc/trilogy/protocol.h

```diff
@@ -547,6 +547,13 @@ typedef struct {
     size_t last_gtid_len;
 } trilogy_ok_packet_t;
 
+typedef struct {
```

Comment:
Adding a reference to https://dev.mysql.com/doc/internals/en/com-stmt-prepare-response.html might be useful here?